Sources/BraveWallet/Crypto/BuySendSwap/BuyProviderSelectionView.swift

```diff
@@ -64,10 +64,10 @@ struct BuyProviderSelectionView: View {
           }
           .padding(.vertical, 10)
         }
+        .listRowBackground(Color(.secondaryBraveGroupedBackground))
       }
     }
     .listStyle(InsetGroupedListStyle())
-    .listBackgroundColor(Color(UIColor.braveGroupedBackground))
```

Comment:
This background colour is still needed. More noticeable in dark mode but the background colour we use has a bit more of a blueish tint to it than iOS default.
<img width="489" alt="Screen Shot 2022-10-27 at 2 23 39 PM" src="https://user-images.githubusercontent.com/5314553/198368941-061cca39-0114-49e2-b279-2cc86b7b0a2f.png">|<img width="489" alt="Screen Shot 2022-10-27 at 2 23 36 PM" src="https://user-images.githubusercontent.com/5314553/198368946-eb72c31f-d053-4376-901c-a7b20c240c64.png">
--|--